common/lib/xmodule/xmodule/modulestore/tests/django_utils.py

```diff
@@ -159,6 +159,23 @@ def xml_store_config(data_dir, source_dirs=None):
 
     return store
 
+
+@patch('xmodule.modulestore.django.create_modulestore_instance')
+def drop_mongo_collections(mock_create):
```

Comment:
Nit: you could just pass `Mock(return_value=None)` as the second argument to `@patch`, and you wouldn't need the argument to `drop_mongo_collections`.